Sources/SwiftInspectorCommands/TypealiasCommand.swift

```diff
@@ -24,33 +24,71 @@
 
 import ArgumentParser
 import Foundation
+import SwiftInspectorCore
 
 final class TypealiasCommand: ParsableCommand {
   static var configuration = CommandConfiguration(
     commandName: "typealias",
     abstract: "Finds information related to the declaration of a typelias"
   )
 
-  @Option()
+  @Option(default: "", help: Help.name)
   var name: String
 
   @Option()
   var path: String
 
   /// Runs the command
   func run() throws {
+    let cachedSyntaxTree = CachedSyntaxTree()
+    let analyzer = TypealiasAnalyzer(cachedSyntaxTree: cachedSyntaxTree)
+    let fileURL = URL(fileURLWithPath: path)
+    let outputArray = try FileManager.default.swiftFiles(at: fileURL)
+      .reduce(Set<String>()) { result, url in
+        let statements = try analyzer.analyze(fileURL: url)
+        let output = filterOutput(statements).map { outputString(from: $0)}
+        return result.union(output)
+    }
+
+    let output = outputArray.filter { !$0.isEmpty }.joined(separator: "\n")
+    print(output)
   }
 
   /// Validates if the arguments of this command are valid
   func validate() throws {
-    guard !name.isEmpty else {
-      throw InspectorError.emptyArgument(argumentName: "--name")
-    }
     guard !path.isEmpty else {
       throw InspectorError.emptyArgument(argumentName: "--path")
     }
     guard FileManager.default.fileExists(atPath: path) else {
       throw InspectorError.invalidArgument(argumentName: "--path", value: path)
     }
   }
+
+  /// Filters the output based on command line inputs
+  private func filterOutput(_ output: [TypealiasStatement]) -> [TypealiasStatement] {
+    guard !name.isEmpty else {
+      return output
+    }
+
+    return output.filter { $0.name == name }
```

Comment:
Should we add a test to handle the case where the same typealias occurs twice in the code you're scanning? This could happen if you are scanning multiple modules of if they have different access control.

Reply:
https://github.com/fdiaz/SwiftInspector/pull/10/commits/dc51dcba4124c745d567d7785379f6e5f868cf78